web/src/components/DatePicker.tsx

```diff
@@ -132,34 +89,35 @@ const CustomDatePicker = ({
 
   return (
     <DateContainer>
-      <StyledInputWrapper>
-        <StyledInput
-          open={isCalendarOpen}
-          value={value}
-          popperPlacement='bottom'
-          selected={isValidJsDate(date?.toJSDate() ?? null) ? date?.toJSDate() : null}
-          onSelect={e => {
-            setValue(DateTime.fromJSDate(e ?? new Date()).toFormat('dd.MM.yyyy'))
-            setDatePickerError('')
+      <LocalizationProvider dateAdapter={CustomAdapterLuxon} adapterLocale={languageCode}>
+        <DatePicker
+          value={date}
```

Comment:
❓ Should we enable `disablePast`? What do you think?

Reply:
it only works for the calendar not for the input field.